script.module.enum34/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.enum34" version="1.1.6" name="enum" provider-name="ethanf">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0" />
+    </requires>
+    <extension library="lib/enum" point="xbmc.python.module" />
```

Comment:
I think it has to be `library="lib"` to work correct, e.g `from enum import Enum`

Reply:
I tested it and you're correct.
I've made the change.